README.md

```diff
@@ -27,9 +25,42 @@ request
     });
 ```
 
+
+### API
+
+---
+
+#### `auth([Object options])` -> `Function`
+
+The export is a factory function which creates the superagent plugin. The
+result of this call is what should be passed into `.use`.
+
+##### Option: scope `String` _(\*:\*\:*)_
+
+You may optionally specify scope(s) desired on the requested token. This option
+is passed directly into the backing [frau-jwt][frau-jwt] library.
```

Comment:
So this allows a page to send a service a limited token, which is cool because than that service can't go completely wild with their account (better than our current setup.)

I think it'd be interesting in the future for the FRA config data to specify the maximum scope, and for the host to also limit what it will honour from the client. That way we can be garunteed like that the grades page doesn't set configs or bobby's more-organized discussion viewer app doesn't read grades, even with XSS. That'd be neat and possibly seamless (key.)

This is a good first step though.